src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+// import classes from '../styles/Nav.module.css';
```

Comment:
- [ ]  Can you clean this commented code to avoid confusion? Also, commented code makes the codes look unclean and unprofessional, I would advise you to always erase commented code.

Reply:
Dear @OLIPLICHE ,

While struggling to link stylings with different methods with `JSX` I have left some 
unnecessary commented codes.

I have now removed the commented codes. Thanks for the thorough review.

Best,

@Zafron047